docs/Maintainer-Guidelines.md

```diff
@@ -105,6 +107,28 @@ is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
 
+### Adding or update formulae
```

Comment:
Should this not say `updating` instead of `update` ?

Reply:
@javian Yeh, it should 👍